x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/metrics/time_dropdown.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiSelect } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  value: number;
+  onChange(event: React.ChangeEvent<HTMLSelectElement>): void;
+}
+
+export const TimeDropdown = (props: Props) => (
+  <EuiSelect
```

Comment:
Can you change this to `compressed="true"` - We'll want to use the compressed form fields inside the panel to save on space.